dagger-functions/src/main/java/io/odpf/dagger/functions/udfs/aggregate/DistinctCount.java

```diff
@@ -32,4 +33,12 @@ public void accumulate(DistinctCountAccumulator distinctCountAccumulator, String
         }
         distinctCountAccumulator.add(item);
     }
+
+    public void merge(DistinctCountAccumulator distinctCountAccumulator, Iterable<DistinctCountAccumulator> it) {
+        for (DistinctCountAccumulator distinctCountAccumulatorInstance : it) {
+            HashSet<String> distinctItems = distinctCountAccumulator.getDistinctItems();
+            distinctItems.addAll(distinctCountAccumulatorInstance.getDistinctItems());
+            distinctCountAccumulator.setDistinctItems(distinctItems);
```

Comment:
instead of calling setDistinctItems() shouldn't it be 

distinctCountAccumulator.getDistinctItems().addAll(distinctCountAccumulator.getDistinctItems())

we should not assign pointer.. `distinctCountAccumulator.getDistinctItems()` it might not create a new object..

Reply:
sure, will change this, thought it will be more explicit